lib/Form/Type/FieldType/TimeFieldType.php

```diff
@@ -44,21 +45,21 @@ public function getParent()
 
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder->addModelTransformer(new FieldValueTransformer($this->fieldTypeService->getFieldType('eztime')));
+        $builder
+            ->addViewTransformer(new FormattedTimeToTimestampTransformer($options['with_seconds']))
+            ->addModelTransformer(new TimeValueTransformer());
```

Comment:
Noob question: what's the difference between Model and View transformers in this case? Based on [Symfony Doc](http://symfony.com/doc/current/form/data_transformers.html#about-model-and-view-transformers) I'd expect both transformers to be Model transformers, but I don't know this that well (or rather at all :stuck_out_tongue_winking_eye:)

Reply:
There are 3 types of data in Symfony forms:
* Model data - used by your model
* Normalized - used by the form type itself
* View data - used as a value in HTML

Model Transformer is doing the conversion between Model<->Norm data.
View Transformer is doing the conversion between Norm<->View data.